FarsiLibrary.Win.DevExpress.17.1/XtraFADateEdit.TouchUI.cs

```diff
@@ -0,0 +1,324 @@
+﻿using System;
```

Comment:
This whole fine is not needed. Add a link to existing file in 15.1.

Reply:
ok